core/src/main/java/io/spine/core/Rejections.java

```diff
@@ -192,4 +193,42 @@ public static boolean isExternal(RejectionContext context) {
         checkNotNull(context);
         return context.getExternal();
     }
+
+    /**
+     * Verifies if the exception was {@linkplain Throwables#getRootCause(Throwable) caused} by
+     * a command rejection.
+     *
+     * @param exception the exception to analyze
+     * @return {@code true} if the exception was created because of a command rejection thrown,
+     *         {@code false} otherwise
+     */
+    public static boolean causedByRejection(Throwable exception) {
+        //TODO:2017-07-26:alexander.yevsyukov: Check against CommandRejection
+        // instead of ThrowableMessage when code generation allows customizing a custom
+        // rejection types instead of `ThrowableMessage`.
+        // See: https://github.com/SpineEventEngine/base/issues/20
+        final Throwable rootCause = Throwables.getRootCause(exception);
+        final boolean result = rootCause instanceof ThrowableMessage;
+        return result;
+    }
+
+    /**
+     * Retrieves the {@linkplain Throwables#getRootCause root cause} of the given {@link Throwable}
+     * as a {@link ThrowableMessage}.
+     *
+     * <p>Throws an {@link IllegalArgumentException} if the root cause is not
+     * a {@code ThrowableMessage}.
+     *
+     * @param throwable the {@link Throwable} wrapping a {@link ThrowableMessage}
+     * @return the wrapped {@link ThrowableMessage}
+     * @throws IllegalArgumentException upon an invalid {@link Throwable}
+     *                                  {@linkplain Throwables#getRootCause root cause}
+     */
+    static ThrowableMessage getCauseRejection(Throwable throwable)
```

Comment:
It's either `getRejectionCause`, where "cause" is a noun. Or just`getCause` — the context is defined by the parent class anyway.